demo/server.js

```diff
@@ -126,28 +141,79 @@ initSqlJs().then(SQL => {
 
   // DATA MANAGEMENT
   function persistData(data) {
-    db.run('INSERT INTO data VALUES (?, ?);', [
+    db.run("INSERT INTO data VALUES (?, ?);", [
       data.transactionHash,
       data.originalData.text,
     ]);
-    fs.writeFile(dbPath, Buffer.from(db.export()), () => { });
+    fs.writeFile(dbPath, Buffer.from(db.export()), () => {});
   }
 
   // Insert a training sample.
-  app.post('/api/data', jsonParser, (req, res) => {
-    const body = req.body
+  app.post("/api/data", jsonParser, (req, res) => {
+    const body = req.body;
     persistData(body);
     return res.sendStatus(200);
   });
 
   // Get original training data.
-  app.get('/api/data/:transactionHash', (req, res) => {
-    const getTextStmt = db.prepare('SELECT text FROM data WHERE transaction_hash == $transactionHash LIMIT 1;');
-    const result = getTextStmt.getAsObject({ $transactionHash: req.params.transactionHash });
+  app.get("/api/data/:transactionHash", (req, res) => {
+    const getTextStmt = db.prepare(
+      "SELECT text FROM data WHERE transaction_hash == $transactionHash LIMIT 1;"
+    );
+    const result = getTextStmt.getAsObject({
+      $transactionHash: req.params.transactionHash,
+    });
     getTextStmt.free();
     const { text } = result;
     res.send({ originalData: { text } });
   });
 
   app.listen(port, () => console.log(`Listening on port ${port}`));
+
+  // ACCURACY RECORD MANAGEMENT
+  function presistAccuracyRecord(accuracy) {
+    db.run("INSERT INTO accuracy VALUES (?, ?, ?, ?,, CURRENT_TIMESTAMP);", [
+      accuracy.transactionHash,
+      accuracy.blockNumber,
+      accuracy.modelId,
+      accuracy.accuracy,
+    ]);
+    fs.writeFile(dbPath, Buffer.from(db.export()), () => {});
+  }
+ // Add a new accuracy record for a model
+  app.post("/api/accuracy", jsonParser, (req, res) => {
+    const body = req.body;
+    presistAccuracyRecord(body);
+    return res.sendStatus(200);
+  });
+
+  // Get the accuracy history for a model
+  function getAccuracyHistory(modelId) {
```

Comment:
Delete this method because it is not used and I don't think it will work so I believe it is misleading.